site/gatsby-site/cypress/e2e/unit/functions/logIncidentHistory.cy.js

```diff
@@ -0,0 +1,81 @@
+const logIncidentHistory = require('../../../../../realm/functions/logIncidentHistory');
+
+const incidentInput = {
+  incident_id: 545,
+  title: 'Chatbot Tessa gives unauthorized diet advice to users seeking help for eating disorders',
+  description:
+    'The National Eating Disorders Association (NEDA) has shut down its chatbot named Tessa after it gave weight-loss advice to users seeking help for eating disorders. The incident has raised concerns about the risks of using chatbots and AI assistants in healthcare settings, particularly in addressing sensitive issues like eating disorders. NEDA is investigating the matter, emphasizing the need for caution and accuracy when utilizing technology to provide mental health support.',
+  reports: [3103, 3104],
+  editors: ['Daniel Atherton'],
```

Comment:
It doesn't make much difference here, but this is a user id now.

Reply:
Fixed